packages/itwin/imodel-react-hooks/src/utils/react-context.ts

```diff
@@ -1,18 +1,15 @@
 /*---------------------------------------------------------------------------------------------
-* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
-* See LICENSE.md in the project root for license terms and full copyright notice.
-*--------------------------------------------------------------------------------------------*/
-
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
 
 import React from "react";
 
 /**
  * like React.createContext, but the default state (the one used when no provider is found by a consumer),
  * throws errors on usage, useful for when consuming without a provider should be considered programmer error.
  */
-export const makeContextWithProviderRequired = <T extends any>(
-  name = "UntitledContext"
-) =>
+export const makeContextWithProviderRequired = <T>(name = "UntitledContext") =>
```

Comment:
this syntax works now? I thought you couldn't make generic arrow functions without an `extends` clause because of ambiguity with the old cast operator in typescript? `<any>5`

Reply:
I don't think they deprecated the old syntax. I'm not sure how the ambiguity can be resolved.
reference: https://stackoverflow.com/questions/32308370/what-is-the-syntax-for-typescript-arrow-functions-with-generics#45576880
maybe do `T extends object`? idk...

Reply:
I do remember the linter rules recommend extending `Record<string, any>` (or with `unknown`), but I guess I wouldn't want the string keys restriction. I would prefer just disabling the lint rule for this line and using `any`. The link I posted above recommends using a hanging comma to disambiguate the syntax.
```tsx
const makeContextWithProviderRequired = <T, >(...args) => {};
```
I would probably vote for disabling the linter rule since I haven't seen the hanging-comma hack before

Reply:
would you rather i just rever the changes to this file?